website/integrations: fix netbird integration

MichaelUray opened the pull request:
<!--
👋 Hi there! Welcome.

Please check the Contributing guidelines: https://docs.goauthentik.io/docs/developer-docs/#how-can-i-contribute
-->

## Details

<!--
Explain what this PR changes, what the rationale behind the change is, if any new requirements are introduced or any breaking changes caused by this PR.

Ideally also link an Issue for context that this PR will close using `closes #`
-->
NETBIRD_AUTH_AUDIENCE was set to `<Cour Client Secret>` instead of `<Your Client ID>`

Added:
`NETBIRD_AUTH_PKCE_DISABLE_PROMPT_LOGIN=true`
[see details here](https://github.com/netbirdio/netbird/issues/3654)


---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make docs`)

MichaelUray committed:
Update index.md

NETBIRD_AUTH_AUDIENCE was set to <Cour Client Secret> instead of <Your Client ID>

Added:
NETBIRD_MGMT_IDP="authentik"
NETBIRD_IDP_MGMT_CLIENT_ID="<PROVIDER_CLIENT_ID>"
NETBIRD_IDP_MGMT_EXTRA_USERNAME="Netbird"
NETBIRD_IDP_MGMT_EXTRA_PASSWORD="<SERVICE_ACCOUNT_PASSWORD>"

Signed-off-by: Michael Uray <michael.uray@gmail.com>

BeryJu reviewed (dismissed):
NETBIRD_IDP_MGMT_EXTRA_USERNAME and NETBIRD_IDP_MGMT_EXTRA_PASSWORD are duplicate with values further up

tanberry commented:
Hi @MichaelUray as @BeryJu pointed out, the two entries you add were already there, on lines 100 and 101. If you want to add them with your format, and your helpful new addition on loines 106 and 107, that's fine, but right now those two are duplicated.

MichaelUray committed:
Update index.md

Removed duplicates
NETBIRD_IDP_MGMT_EXTRA_USERNAME
NETBIRD_IDP_MGMT_EXTRA_PASSWORD
from pull request.

Signed-off-by: Michael Uray <michael.uray@gmail.com>

MichaelUray commented:
>  If you want to add them with your format, and your helpful new addition on loines 106 and 107, that's fine, but right now those two are duplicated.

Hi, thanks, I think I corrected it. Is there anything else to do for me?
(I am not very familiar with the Github workflow)

tanberry reviewed (approved):
Looks great, thanks so much @MichaelUray !

dominic-r commented on website/integrations/networking/netbird/index.md:
```diff
@@ -92,13 +92,16 @@ NETBIRD_AUTH_OIDC_CONFIGURATION_ENDPOINT="https://authentik.company/application/
 NETBIRD_IDP_MGMT_EXTRA_USERNAME="Netbird"
 NETBIRD_IDP_MGMT_EXTRA_PASSWORD="<Your Service Account password>"
+
+# needs disabling due to issue with IdP. Learn more [here](https://github.com/netbirdio/netbird/issues/3654)
```
```suggestion
# needs disabling due to issue with IdP. Learn more at https://github.com/netbirdio/netbird/issues/3654
```

Markdown links have no effect in code blocks and should thus be removed to improve readability

dominic-r reviewed (approved):
Thanks!

dewi-tik committed:
Update website/integrations/networking/netbird/index.md

Co-authored-by: Dominic R <dominic@sdko.org>
Signed-off-by: Dewi Roberts <dewi@goauthentik.io>
